test/test_files/projection/skip_limit.test

```diff
@@ -2,6 +2,46 @@
 
 --
 
+-CASE LoadWithExpressionSkip
+-STATEMENT LOAD FROM '${KUZU_ROOT_DIRECTORY}/dataset/tinysnb/vPerson.csv' RETURN * SKIP 4.1;
+---- error
+Runtime exception: The number of rows to skip/limit must be a non-negative integer.
+
+-STATEMENT LOAD FROM '${KUZU_ROOT_DIRECTORY}/dataset/tinysnb/vPerson.csv' RETURN * SKIP -4;
+---- error
+Runtime exception: The number of rows to skip/limit must be a non-negative integer.
+
+-STATEMENT LOAD FROM '${KUZU_ROOT_DIRECTORY}/dataset/tinysnb/vPerson.csv' RETURN * SKIP 'abc';
+---- error
+Runtime exception: The number of rows to skip/limit must be a non-negative integer.
+
+-STATEMENT LOAD FROM '${KUZU_ROOT_DIRECTORY}/dataset/tinysnb/vPerson.csv' RETURN * SKIP 2*2;
```

Comment:
What is the difference between your test and the one in this file?
```
-LOG SkipWithConstantExpr
-STATEMENT MATCH (a:person) RETURN a.ID SKIP 1 + 2
---- 5
5
7
8
9
10
```
Both of them are testing constant expression as skip number